devenv/README.md

```
@@ -0,0 +1,225 @@
+                    │  │      :6379       │                  │
+                    │  └──────────────────┘                  │
+                    │  ┌──────────────────┐                  │
+                    │  │    Registry      │                  │
+                    │  │     :50000       │                  │
+                    │  └──────────────────┘                  │
+                    └────────────────────────────────────────┘
+```
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)